src/WS.cpp

```diff
@@ -232,14 +263,20 @@ enum
     PNT_STREAM2_JPEG_ENABLED = 1,
     PNT_STREAM2_JPEG_PATH,
     PNT_STREAM2_JPEG_QUALITY,
-    PNT_STREAM2_JPEG_REFRESH
+    //PNT_STREAM2_JPEG_REFRESH,
+    PNT_STREAM2_JPEG_CHANNEL,
+    PNT_STREAM2_STATS,
+    PNT_STREAM2_FPS
 };
 
 static const char *const stream2_keys[] = {
     "jpeg_enabled",
     "jpeg_path",
     "jpeg_quality",
-    "jpeg_refresh"};
+    //"jpeg_refresh",
```

Comment:
Remove this line instead?

Reply:
will be done in future, if i'am really sure we do not need jpeg_refresh.
This value now is replaced by fps.